server.js

```diff
@@ -19,11 +36,50 @@ app.use(cors())
 app.use(bodyParser.json())
 
```

Comment:
Also add the error handling for the database connection. 
Check Damiens video about setting that middleware up here:
https://technigo.wistia.com/medias/l5wqrcu4to?wtime=9m22s